models/org.go

```diff
@@ -418,6 +448,12 @@ func GetOwnedOrgsByUserIDDesc(userID int64, desc string) ([]*User, error) {
 	return getOwnedOrgsByUserID(x.Desc(desc), userID)
 }
 
+// GetOrgsCanCreateRepoByUserIDDesc returns a list of organizations where given user ID
+// are allowed to create repos, ordered descending by the given condition.
+func GetOrgsCanCreateRepoByUserIDDesc(userID int64, desc string) ([]*User, error) {
+	return getOrgsCanCreateRepoByUserIDDesc(x.Desc(desc), userID)
```

Comment:
Why does `getOrgsCanCreateRepoByUserIDDesc` is needed if it is only called from one place, also `desc string` is also always used `updated_unix` so parameter can be removed and just added to query above.
Also it should probably be renamed to `GetOrgsCanCreateRepoByUserID`

Reply:
changed